packages/components/addon/components/hds/side-nav/button.hbs

```diff
@@ -0,0 +1,14 @@
+<Hds::Interactive
+  class="hds-side-nav__button"
+  @current-when={{@current-when}}
+  @models={{hds-link-to-models @model @models}}
+  @query={{hds-link-to-query @query}}
+  @replace={{@replace}}
+  @route={{@route}}
+  @isRouteExternal={{@isRouteExternal}}
+  @href={{@href}}
+  @isHrefExternal={{@isHrefExternal}}
+  ...attributes
```

Comment:
@KristinLBradley if you look at the `Hds::Button`, you will see that when it's icon-only then a `<span>` containing required `@text` is displayed for screen-readers only. please sync with @MelSumner if she prefers this approach here too, or a (required) `aria` attribute is enough to describe the button for screen-readers

Reply:
It looks like `Hds::Button` just uses an aria-label to include text when it's icon-only unless I'm looking at it wrong...

Should I make the aria-label required? (I think it should be. Or else a hidden span with accessible text.)

Reply:
Fixed